cpp/CMakeLists.txt

```diff
@@ -17,6 +17,9 @@ set(CXX_SRC
 # Add RRTMGP as a library and make sure we link against YAKL
 add_library(rrtmgp STATIC ${CXX_SRC})
 target_link_libraries(rrtmgp yakl)
+if (RRTMGP_ENABLE_KOKKOS)
```

Comment:
For the sake of rrtmgp as a standalone package, maybe we should expose this var as a CACHE var in the main CMakeLists.txt, and also add a `find_package` call. That way, if some other package wants to use rrtmgp with kokkos, they at least have a somewhat well defined (in a CMake sense) way of triggering it.

Reply:
The cmake situation in rrtmgp is a little weird. `cpp/test` is a subdirectory of `cpp` but the CMakeLists.txt in `test` is the one that sets up the project and adds YAKL and Kokkos. It's clear that `cpp/CMakeLists.txt` only expects to be used by an external project that has already set up YAKL and Kokkos.